assets/carousel.js

```diff
@@ -505,14 +518,25 @@ class Carousel {
     }
   }
 
-  getPrevSibling(element, selector) {
-    if (element) {
-      let sibling = element.previousElementSibling;
+  getSiblingElement(element, selector, direction) {
+    if (!element) return false;
 
-      while (sibling) {
-        if (sibling.matches(selector)) return sibling;
-        sibling = sibling.previousElementSibling;
-      }
+    let sibling;
+
+    const setSibling = () => {
+      sibling = direction === this.event.prev
+                ? element.previousElementSibling
+                : direction === this.event.next
+                ? element.nextElementSibling
+                : false
```

Comment:
this is hard to understand. I think this is better:

```suggestion
      let sibling = false;

      if (direction === this.event.prev) {
        sibling = element.previousElementSibling;
      } else if (direction === this.event.next) {
        sibling = element.nextElementSibling;
      }
```

Reply:
Yeah, it was slightly convoluted